### DISCUSSION TITLE
Filter values before load csv file using 'tables' and 'table-reader' plugins.

### DISCUSSION THREAD

**Discussion by Marlony-Gabriel** [mkdocs/mkdocs#3393] [Q&A]
Hello. Good morning/afternoon/night everyone.

I'm using the table-reader plugin to load a file and show the data in my project.
Now, I'm trying to find a way to filter what values will be showed before load the table.
Is that possible ?
I mean, is there a way I can ask the person who is acessing the page to fill a field value for example, and load only the lines within the value provide ?

**Comment by pawamoy** [comment]
I think you should ask this to the author of the table-reader plugin :)

**Reply by Marlony-Gabriel** [reply]
Hi @pawamoy , thank you for your answer.
I try to find a way to contact him, but there is no "Discussion" section in his github page. I'm not so good with github yet but now I saw there's a email in his profile. I'll try to contact him. Thank you.

**Reply by Andre601** [reply]
Usually issues can be used as a alternative to discussions to ask questions. In fact before discussions was a thing were issues the main place for questions in most projects.
